CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 Mapbox welcomes participation and contributions from everyone.
 
+# 10.0.2 December 13, 2021
+
+## Bug fixes 🐞
+* Fix billing issue when upgrading Mapbox Maps SDK from v9 to v10.
+
+## Dependencies
+* Bump common to v20.0.3.
+
 # 10.0.1 November 26, 2021
```

Comment:
we want to leave 10.0.1 changelog here, correct?

Reply:
yeah, we will keep it as a notification on why the older patch doesn't work as expected. cc @pengdev

Reply:
do we need to update changelog entry for 10.0.1 then? cause right now we have 2 similar patches per changelog while only latest one will be available as a binary

Reply:
I don't think we need to change the entry for 10.0.1 as it was mentioned that `a new patch will be issued shortly` in the changelog. so this one will work as expected.